Code/GraphMol/MolDraw2D/MolDraw2DQt.cpp

```diff
@@ -27,8 +27,7 @@ MolDraw2DQt::MolDraw2DQt(int width, int height, QPainter &qp, int panelWidth,
 // ****************************************************************************
 void MolDraw2DQt::setColour(const DrawColour &col) {
   MolDraw2D::setColour(col);
-  QColor this_col(int(255.0 * col.get<0>()), int(255.0 * col.get<1>()),
-                  int(255.0 * col.get<2>()));
+  QColor this_col(int(255.0 * col.r), int(255.0 * col.g), int(255.0 * col.b));
```

Comment:
I'm assuming that MolDraw2Qt drops the alpha channel?

Reply:
was an oversight since I don't use the Qt thing